lib/instrument/headers.js

```diff
@@ -0,0 +1,260 @@
+import {archive, unarchive} from './transformer/nskeyed';
+import plistlib from 'bplist-parser';
+
+const FLAGS_TYPE = 2;
+const DTX_MESSAGE_HEADER_LENGTH = 32;
+const DTX_MESSAGE_PAYLOAD_HEADER_LENGTH = 16;
+const DTX_MESSAGE_HEADER_MAGIC = 0x1F3D5B79;
+
+class DTXMessageHeader {
+  static build (data) {
```

Comment:
could you please document all methods and data structures like it is done in other files?